nsqadmin/static/js/views/counter.js

```diff
@@ -0,0 +1,109 @@
+var _ = require('underscore');
+var $ = require('jquery');
+
+var AppState = require('../app_state');
+
+var BaseView = require('./base');
+
+var CounterView = BaseView.extend({
+    className: 'counter container-fluid',
+
+    template: require('./counter.hbs'),
+
+    initialize: function() {
+        BaseView.prototype.initialize.apply(this, arguments);
+        this.listenTo(AppState, 'change:graph_interval', this.render);
+        this.start();
+    },
+
+    start: function() {
+        this.looping = false;
+        this.frameRate = 50;
+        this.targetPollInterval = 30000;
+        this.currentNum = -1;
+        this.tries = 0;
+        this.updateStats();
+        this.startLoop(100);
+    },
+
+    startLoop: function(i) {
+        this.interval = i;
+        this.poller = setInterval(this.updateStats.bind(this), i);
+    },
+
+    updateStats: function() {
+        $.get('/counter').done(function(data) {
```

Comment:
This code should prob live in a model, but again, this is more "future-proofing". If we're gonna scrap and move to React, it might not be a big deal to just leave this as is.